packages/react-notifications/src/components/notification/notification.tsx

```diff
@@ -40,42 +40,40 @@ const NotificationIcon = (props: PropsIcon) => {
 };
 NotificationIcon.displayName = 'NotificationIcon';
 
-const defaultProps = {
-  fixed: false,
-};
-type DefaultProps = typeof defaultProps;
 export type Props = {
   domain: TAppNotificationDomain;
   type: TAppNotificationKind;
   fixed: boolean;
   onCloseClick: (event: React.SyntheticEvent) => void;
   children: React.ReactNode;
-} & DefaultProps;
+};
+const defaultProps: Pick<Props, 'fixed'> = {
+  fixed: false,
+};
 
-const Notification = (props: Props) => (
-  <div css={getStylesForNotification(props)} {...filterDataAttributes(props)}>
-    <div css={getStylesForContent(props)}>{props.children}</div>
-    {props.onCloseClick ? (
-      <div>
-        <FormattedMessage {...messages.hideNotification}>
-          {label => (
```

Comment:
About this, the argument returned by `FormattedMessage` has this signature

```ts
children?(...nodes: React.ReactNodeArray): React.ReactNode;
```

Which is a bit surprising, but at the same time we can just use `intl.formatMessage` since we need to pass a `string`.